Service/Response/ResponseStrategyTransformer.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Auto1\ServiceAPIClientBundle\Service\Response;
+
+use Auto1\ServiceAPIClientBundle\Service\ResponseTransformerStrategy;
+use Auto1\ServiceAPIComponentsBundle\Exception\Core\ConfigurationException;
+use Auto1\ServiceAPIComponentsBundle\Service\Endpoint\EndpointRegistryInterface;
+use Auto1\ServiceAPIRequest\ServiceRequestInterface;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Log\LoggerAwareInterface;
+use Psr\Log\LoggerAwareTrait;
+use Psr\Log\NullLogger;
+
+class ResponseStrategyTransformer implements LoggerAwareInterface, ResponseTransformerInterface
```

Comment:
there is no use to add logging capabilities to this class, better to remove the LoggerAwareInterface

Reply:
removed